homeassistant/components/konnected/switch.py

```diff
@@ -130,6 +131,15 @@ def _set_state(self, state):
             state,
         )
 
+    async def async_set_state(self, state):
```

Comment:
This can be a callback since we don't await inside.
```suggestion
    @callback
    def async_set_state(self, state):
```